src/test/java/stepDefinition/BookNowMenuLevelSteps.java

```diff
@@ -0,0 +1,73 @@
+package stepDefinition;
+
+import cucumber.TestContext;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.BookNowPage;
+import utils.SeleniumActions;
+
+public class BookNowMenuLevelSteps {
+    TestContext testContext;
+    WebDriver driver;
+
+    public BookNowMenuLevelSteps(TestContext context) {
+        testContext = context;
+        driver = testContext.getWebDriverManager().getDriver();
+    }
+
+
+    @Then("User validates booking form is displayed")
+    public void user_validates_content_form_is_displayed() {
+        WebElement bookNowContentPage = driver.findElement(BookNowPage.BookingForm);
+        Assert.assertTrue(bookNowContentPage.isDisplayed());
+    }
+
+    @When("The user clicks the input grid for guest information with value{string} and enters guest information {string}")
+    public void theUserClicksTheInputGridForGuestInformationWithValueName(String inputGrid, String guestInfo) {
+        WebElement guestInfoInput = driver.findElement(BookNowPage.GuestDetailInput(inputGrid));
+        guestInfoInput.click();
```

Comment:
why are you not using the seleniumAction click?

Reply:
Done